app/containers/UIKit/MessageBlock.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/prop-types */
```

Comment:
`blocks` is a typical `any` object.
It's safe to add it as `any` on prop types.